dev/src/Command/SplitCommand.php

```diff
@@ -157,9 +163,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $releaseNotes = new ReleaseNotes($changelog);
+        $branchToPush = null;
 
         if ($componentId = $input->getOption('component')) {
-            $components = [new Component($componentId)];
+            $branchToPush = $input->getOption('component-branch');
+            $components = [new Component($componentId, null)];
```

Comment:
revert unneeded changes 
```suggestion
            $components = [new Component($componentId)];
```